python/pyspark/ml/feature.py

```diff
@@ -775,7 +775,7 @@ def mean(self):
 
 
 @inherit_doc
-class StringIndexer(JavaEstimator, HasInputCol, HasOutputCol):
+class StringIndexer(JavaEstimator, HasInputCol, HasOutputCol, HasHandleInvalid):
```

Comment:
Some other classes has no shared accessor class (e.g. `StandardScaler`, `RegexTokenizer`) corresponding to arbitrary properties. It might be better to keep `handleInvalid` only inside of `StringIndexer` or create other shared accessor classes for `StandardScaler` etc to standardize the way of accessing to arbitrary properties.